cd.yaml

```diff
@@ -4,4 +4,4 @@
 
 staging:
   vars:
-    API_VERSION: master
+    API_VERSION: nferx
```

Comment:
we have phased out this deployment system. merge the appengine changes in the rest-api branch and then use the `nferx-dot-open-targets-eu-dev.appspot.com` google appengine endpoint instead.